src/components/Menu/Menu.tsx

```diff
@@ -39,7 +39,8 @@ export type MenuItemsProps = ExtractProps<typeof HeadlessMenu.Items>;
  * a profile menu with links to settings) or a set of actions
  */
 export const Menu = (props: MenuProps) => {
-  return <HeadlessMenu {...props} />;
+  const menuClassNames = clsx(props.className, styles['menu']);
+  return <HeadlessMenu as="div" className={menuClassNames} {...props} />;
```

Comment:
Since you haven't extracted `className` out of `props`, would it be applied again with `{...props}`?

Reply:
Good call, it would. i can extract that one out to prevent doubling